### ISSUE TITLE
Update config file docs

### ISSUE THREAD

**Issue by imbsky** [dependabot/dependabot-core#1461]
This page seems to be missing information about `github_actions`.

https://dependabot.com/docs/config-file

**Comment by feelepxyz** [comment]
@imbsky thanks for raising this, we've currently disabled actions support due to a bug that prevented Apps from modifying the workflow file, but might be able to re-enable now that it's fixed.

**Comment by imbsky** [comment]
Oh, I see!

**Comment by imbsky** [comment]
@feelepxyz When re-enable actions support, could you please update documentation and close this issue?

**Comment by feelepxyz** [comment]
@imbsky it's still a work in progress so not won't create any PRs yet. Hopefully we'll get this wrapped up in the next month or so.

**Comment by imbsky** [comment]
Great!

**Comment by imbsky** [comment]
Is this still disabled?

**Comment by feelepxyz** [comment]
@imbsky yeah sadly! There's new app permission that needs to be added which hasn't been prioritised yet. Waiting for an update on it.

**Comment by imbsky** [comment]
Okay, I'll wait.

**Comment by imbsky** [comment]
Any updates?

**Comment by feelepxyz** [comment]
@imbsky I'm chasing internally. Still need the new app permission to be deployed.

**Comment by imbsky** [comment]
@feelepxyz I'm both an Actions user and an outside contributor who has made some improvements, if Actions is supported by Dependabot, I don't have to manually create a lot of PR, so could you to enable it as soon as possible? I don't want to bother you, but please.

**Comment by feelepxyz** [comment]
We're waiting on an internal change to actions allowing any app to edit workflow files. It's not specific to Dependabot.

**Comment by imbsky** [comment]
Oh, I see. That sounds to take some time.

**Comment by LeoColomb** [comment]
🎉 https://github.blog/changelog/2020-04-07-github-apps-workflow-permission-2/

**Comment by peaceiris** [comment]
Correct URL is https://github.blog/changelog/2020-04-07-github-apps-workflow-permission/ 🎉

**Comment by staticdev** [comment]
It seems like something is still wrong. I have github_actions enabled, but it is not creating pull requests yet. I see in dependabot logs that it finds newer versions for actions though.

**Comment by imbsky** [comment]
This is not yet enabled. So nothing is wrong, but I hope it will be enabled asap.

**Comment by feelepxyz** [comment]
We've just launched a beta of Dependabot natively integrated into GitHub that supports updating Actiosn workflow files: https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

Config file docs: https://help.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates

FYI - the new version currently doesn't support private git dependencies. We're working on adding this over the next few months.

**Comment by peaceiris** [comment]
It works! Great 🎉 

https://github.com/peaceiris/actions-gh-pages/pull/334

<img width="609" alt="Screen Shot 2020-06-03 at 21 50 56" src="https://user-images.githubusercontent.com/30958501/83638835-bc63b900-a5e4-11ea-8e8f-f71c724c81c0.png">


I will leave my feedback. (I do not know whether here is the right place for feedback or not.)

## Please detect pre-release and do not bump it

The [actions/setup-node@v2.0.0](https://github.com/actions/setup-node/releases/tag/v2.0.0) is a beta version (pre-release). We need a useful filter for not bumping it.

## `open-pull-requests-limit` does not work

I setted `open-pull-requests-limit: 1` like the following but the dependabot has opened the two pull-requests at the same time.

- https://github.com/peaceiris/actions-gh-pages/pull/334
- https://github.com/peaceiris/actions-gh-pages/pull/333

```yaml
- package-ecosystem: "github-actions"
  directory: "/"
  schedule:
    interval: "daily"
  open-pull-requests-limit: 1
  labels:
  - "dependencies:ci"
  commit-message:
    prefix: ci
```

https://github.com/peaceiris/actions-gh-pages/blob/65dc7af0847a52d3e8dfa56f3f64dcb0acf6032f/.github/dependabot.yml#L28

**Comment by staticdev** [comment]
@feelepxyz I don't see `Security & analysis tab` on `Settings` from repositories I own and have dependabot-preview (as mentioned in https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/):

![image](https://user-images.githubusercontent.com/1446633/83640510-97d6f500-a582-11ea-91d1-1677d7d00b07.png)

**Comment by feelepxyz** [comment]
@peaceiris nice one, will take a look at the open pull request limit not being honoured.

> Please detect pre-release and do not bump it

Yep, it's on our list of improvements. Will see what metadata we can get from tags.

@staticdev ah yes this page hasn't been fully rolled out yet. You can enable it from the `Security` tab on the repository under `Dependabot alerts` > `Dependabot Security Updates`.

**Comment by staticdev** [comment]
I've enabled Security Updates in 3 public repos of mine, then I entered dependabot.com and selected `Update config file` and it didn't work. When I click it again I see the message: `Something went wrong when creating your Pull Request. Try again or contact support.`.

**Comment by feelepxyz** [comment]
@staticdev thanks for reporting, looks like we've tried to generate an invalid config file. Will take a look at fixing this tomorrow 👌

**Comment by staticdev** [comment]
@feelepxyz if you want the repos to check:
* https://github.com/staticdev/pdf-split-tool
* https://github.com/staticdev/toml-validator
* https://github.com/staticdev/django-sorting-bootstrap

**Comment by jurre** [comment]
Thanks for reporting this @staticdev, we've fixed some issues and I _think_ you should be able to upgrade now, could you try again?

**Comment by staticdev** [comment]
@jurre Good news! Dependabot could now create the pull requests, and changed the name/path of dependabot config file.

One strange thing I noticed is that, all of the repositories I tested gave me this warning:
"You're using unsupported features

The new version does not yet support private git dependencies. If you use these we recommend leaving Dependabot Preview active."

I am curious to know what unsupported features since I am not using private repositories...

**Comment by jurre** [comment]
You should be all good in that case 👍 we can't detect if the git dependencies you're using are private or not, so if you're using any git dependencies we show that warning. I'll think about the wording if we can make that more clear

**Comment by staticdev** [comment]
@feelepxyz I have 3 other questions regarding this new config, maybe you could help me out:
* Can the open-pull-requests-limit setting be omitted, to go with the default?
* Any way to avoid hardcoding schedule.time? What is this based on, anyway? I got different times when upgrading projects.
* Do you know why dependabot badge now shows inactive on upgraded projects? Is this a bug? 

Thanks again.

**Comment by feelepxyz** [comment]
> * Can the open-pull-requests-limit setting be omitted, to go with the default?

Yup, the default is 5

> Any way to avoid hardcoding schedule.time? What is this based on, anyway? I got different times when upgrading projects.

This should be set from your account settings in your [dependabot dashboard](https://app.dependabot.com/). You can omit it and the default 5am UTC will be used.

> Do you know why dependabot badge now shows inactive on upgraded projects? Eg. here. Is this a bug?

Yup, this is broken with the updated and we're tracking this here: https://github.com/dependabot/feedback/issues/968

**Comment by staticdev** [comment]
> > * Can the open-pull-requests-limit setting be omitted, to go with the default?
> 
> Yup, the default is 5
> 
> > Any way to avoid hardcoding schedule.time? What is this based on, anyway? I got different times when upgrading projects.
> 
> This should be set from your account settings in your [dependabot dashboard](https://app.dependabot.com/). You can omit it and the default 5am UTC will be used.
> 
> > Do you know why dependabot badge now shows inactive on upgraded projects? Eg. here. Is this a bug?
> 
> Yup, this is broken with the updated and we're tracking this here: [dependabot/feedback#968](https://github.com/dependabot/feedback/issues/968)

Thanks again!!

**Comment by infin8x** [comment]
Closing this as the original issue (lack of `github_actions` documentation) is fixed: https://help.github.com/en/github/administering-a-repository/keeping-your-actions-up-to-date-with-github-dependabot
